.github/workflows/node-ci.yml

```diff
@@ -183,3 +183,7 @@ jobs:
         full: yarn test
         limited-trigger: ${{ steps.js-ts-files.outputs.filtered }}
         limited: yarn test --findRelatedTests --passWithNoTests {}
+
+    - name: Mocha (eslint-plugin-khan)
+      working-directory: ./packages/eslint-plugin-khan
+      run: yarn test
```

Comment:
https://typescript-eslint.io/packages/rule-tester/#vitest shows how to use `vitest` with `@typescript-eslint/rule-tester`.  I'm going to give that a try, but using jest, in a future PR.